app/src/main/res/layout/item_selection_interaction_items.xml

```diff
@@ -0,0 +1,29 @@
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto">
+  <data>
+    <variable
+      name="htmlContent"
+      type="String"/>
+  </data>
+  <RelativeLayout
+    android:id="@+id/checkbox_container"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_margin="4dp"
+    android:orientation="vertical">
+    <androidx.appcompat.widget.AppCompatCheckBox
```

Comment:
As discussed earlier, this can be a CheckBox or RadioButton depending upon the values in `customization_args`

Reply:
This is handled in different view. For reusability I am making use of MultipleChoiceInput View_TYPE.